app/renderer/ui/containers/WalletForm.tsx

```diff
@@ -0,0 +1,365 @@
+import * as React from "react"
+import { Route, RouteComponentProps, Switch } from "react-router"
+import { connect } from "react-redux"
+import { bindActionCreators, Dispatch } from "redux"
+
+import log from "app/common/logging"
+import { assertNever } from "app/common/utils"
+import { StoreState } from "app/renderer/store"
+import { Wallets, newMnemonicsErrors } from "app/common/runtimeTypes/storage/wallets"
+import {
+  encryptWalletErrors,
+  importSeedErrors as validteMnemonicsErrors,
+} from "app/common/runtimeTypes/ipc/wallets"
+import { newMnemonics, encryptWallet, validateMnemonics } from "app/renderer/api"
+import { IAction } from "app/renderer/actions/helpers"
+import { saveWallet } from "app/renderer/actions/loginForm"
+import { PATHS as MAIN_CONTAINER_PATHS } from "app/renderer/ui/containers/Main"
+
+import { WalletForm } from "app/renderer/ui/components/walletForm"
+import {
+  WalletEncryptionPassword,
+  WalletSeedBackup,
+  WalletSeedConfirmation,
+  WalletSeedTypeSelection,
+  Welcome
+} from "app/renderer/ui/components/walletForm/steps"
+
+/**
+ * Props that match redux store state
+ *
+ * @export
+ * @interface StateProps
+ */
+export interface StateProps {
+  wallets: Wallets,
+}
+
+/**
+ * Props that match redux actions
+ *
+ * @export
+ * @interface DispatchProps
+ */
+export interface DispatchProps {
+  actions: any
+}
+
+/**
+ * Props that contain configuration
+ *
+ * @interface Props
+ */
+interface Props {
+  services: any
+}
+
+/**
+ * Function to map the dispatch of actions to props
+ * @param dispatch
+ */
+const mapDispatchToProps = (dispatch: Dispatch<IAction>): DispatchProps => {
+  return {
+    actions: bindActionCreators({ saveWallet }, dispatch)
+  }
+}
+
+/**
+ * Function to map store state to props
+ * @param state
+ */
+const mapStateToProps = (state: StoreState): StateProps => {
+  return ({
+    wallets: state.wallets,
+  })
+}
+
+export const PATHS = {
```

Comment:
Same thing here, URLs are to be defined in [constants](https://github.com/witnet/sheikah/blob/master/app/renderer/constants/urls.ts)